tools/tests/components.yaml

```
@@ -2,39 +2,77 @@ openfoam-adapter:
   repository: https://github.com/precice/openfoam-adapter
   template: component-templates/openfoam-adapter.yaml
   build_arguments: # these things mean something to the docker-service
+    PRECICE_REF:
+      description: Version of preCICE to use
+      default: "main"
+    PLATFORM:
+      description: Dockerfile platform used
+      default: "ubuntu_2204"
+    TUTORIALS_REF:
+      description: Tutorial git reference to use
+      default: "master"
     OPENFOAM_EXECUTABLE:
-      options: ["openfoam2112"]
+      options: ["openfoam2306","openfoam2212",openfoam2112]
```

I guess we need the quotes everywhere:

```suggestion
      options: ["openfoam2306","openfoam2212","openfoam2112"]
```